SenderEmail/pom.xml

```diff
@@ -47,6 +47,27 @@
 			<version>2.2.0</version>
 		</dependency>
 		<!--  ## End Secret Enviroment ##  -->
+
+<!--	## Start  Aspect Oriented  Programming  ## -->
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context</artifactId>
+			<version>6.1.1</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-aop</artifactId>
+			<version>6.1.1</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.aspectj</groupId>
+			<artifactId>aspectjweaver</artifactId>
+			<version>1.9.20</version>
+		</dependency>
+
```

Comment:
You can replace all these dependencies with one dependency and still make it work.
```xml
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-aop</artifactId>
        </dependency>
```